genutil/common.go

```diff
@@ -310,19 +330,23 @@ func FindAllChildren(e *yang.Entry, compBehaviour CompressBehaviour) (map[string
 	// be direct children of the entity representing e.
 	var errs []error
 	// prioNames is the set of names under the prioritized data container
-	// that are added as children. This is a whitelist for any duplicate
+	// that are added as children. This is a whitelist for any shadowed
 	// names in the deprioritized data container.
 	prioNames := map[string]bool{}
 	// directChildren is used to store the nodes that will be mapped to be direct
 	// children of the struct that represents the entry e being processed. It is
 	// keyed by the name of the child YANG node ((yang.Entry).Name).
 	directChildren := make(map[string]*yang.Entry)
+	// shadowChildren is are direct children that are duplicates of direct
+	// childrens that have the same name and de-prioritized due to the
```

Comment:
`that are duplicates of direct children`

Reply:
Reworded.